src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+import Image, { ImageProps } from '../Image'
+
+interface NavbarProps {
+  /** Array of JSX elements */
+  links?: JSX.Element[]
+  /** custom style applied to individual links */
+  linkStyle?: string
+  /** Logo can be text or an image */
+  logo: string | HTMLImageElement
+  /** Logo link */
+  logoURL: string
+  // /** Pins navbar (or not) */
+  sticky?: boolean
+  /** custom style applied to the navbar */
+  style?: string
+}
+
+class Navbar extends React.Component<NavbarProps> {
+  public static defaultProps = {
+    logoURL: '/',
+    sticky: false,
+  }
+
+  public render() {
+    const sticky = css`
+      position: sticky;
+      top: 0;
+    `
+
+    const navbarStyle = css`
+      box-shadow: 0 2px 2px -2px rgba(0, 0, 0, 0.15);
+      padding: 10px;
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+      background-color: white;
+      ${this.props.style};
+      ${this.props.sticky ? sticky : ''};
+    `
+    const linkStyle = css`
+      padding-right: 3vw;
+      color: #65696c;
+      font-family: Helvetica, sans-serif;
```

Comment:
let's not do font stuff here, that should be up to each individual flatpage

Reply:
k removing lines 43-45

Reply:
jk, 44&45